app/enums/survey_question_type.rb

```diff
@@ -2,6 +2,14 @@ class SurveyQuestionType < ClassyEnum::Base
   def requires_answer_list?
     false
   end
+
+  def fields
+    []
+  end
+
+  def all_fields
+    [:agree_text]
+  end
```

Comment:
@cgratigny : I'm a little concerned with keeping this updated when we add a new field to one of the other types (that's the kind of thing that causes the MaritalStatus issue we just had). Since this is the sum of all the fields of all the types, how about something like `SurveyQuestionType.all.map{|type| type.fields}.flatten.uniq`?

Reply:
Great point.